airbyte-integrations/connectors/source-confluence/source_confluence/source.py

```diff
@@ -0,0 +1,126 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+
+from abc import ABC
+from base64 import b64encode
+from typing import Any, Dict, Iterable, List, Mapping, MutableMapping, Optional, Tuple
+
+import requests
+from airbyte_cdk.sources import AbstractSource
+from airbyte_cdk.sources.streams import Stream
+from airbyte_cdk.sources.streams.http import HttpStream
+from airbyte_cdk.sources.streams.http.requests_native_auth import TokenAuthenticator
+from airbyte_cdk.sources.utils.transform import TransformConfig, TypeTransformer
```

Comment:
is this needed ?

Reply:
I don't see it being used anywhere can you remove this

Reply:
^ @tuanchris

Reply:
This is needed to normalize the data according to the schema. Removing this gives the following error: 
![image](https://user-images.githubusercontent.com/52090179/140485005-b2a438f5-391f-42ec-b7d3-068be3a9f56a.png)